.github/workflows/ci.yml

```diff
@@ -258,34 +258,19 @@ jobs:
           - target: cross-arm64
             compiler: gcc
             host_os: ubuntu-24.04
-          - target: cross-arm64-amalgamation
-            compiler: gcc
-            host_os: ubuntu-24.04
           - target: cross-ppc64
             compiler: gcc
             host_os: ubuntu-24.04
-          - target: cross-riscv64
-            compiler: gcc
-            host_os: ubuntu-24.04
-          - target: cross-s390x
-            compiler: gcc
-            host_os: ubuntu-24.04
```

Comment:
Not sure about this one. Isn't it the only platform on big endian? Would be good to catch those before merging things.

Reply:
Ah yeah you're right. Let's see if MIPS64 is faster here #4167

Reply:
Time to run the tests under QEMU

S390 847 seconds
MIPS64 357 seconds

8 minutes faster :grimacing: 

So I would propose bumping S390 to nightly and keeping MIPS64 as our big endian test.